service/src/test/java/uk/gov/hmcts/reform/fpl/utils/NotifyAdminOrderIssuedTestHelper.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.hmcts.reform.fpl.utils;
```

Comment:
I am a firm believer that private methods for tests should belong in the test class they belong to

Reply:
Multiple controller tests make use of these private methods, this seems better than duplicating for each controller test